### ISSUE TITLE
Item-specific forms

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#566]
What if certain forms could be added to attendees or to the checkout process based on items (e.g, a form for T-Shirts asking for a color or a form for a scholarship pass asking to explain why you should have a scholarship),

**Comment by harrislapiroff** [comment]
Might alleviate _some_ of the demand for #428

**Comment by dadamson** [comment]
Yes please! I'm going to fake this for now with a text-area form (and instructions to add T-shirt details therein), but it would be super useful to be able to have a detailed form for an item, or to have form-like options within the item itself.
